scripts/test-helm.sh

```diff
@@ -1,95 +1,150 @@
 
 USAGE="
 Usage:
-  $(basename "$0") <service> <release_version>
+  $(basename "$0") <service>
 
 <service> should be an AWS service for which you wish to run tests -- e.g.
 's3' 'sns' or 'sqs'
 
-<release_version> should be the SemVer version string to build a Helm chart
-for. This release version string should match the name of the Docker image that
-has previously been built.
-
 Environment variables:
-  PRESERVE:                 Preserve kind k8s cluster for inspection (<true|false>)
-                            Default: false
-  TMP_DIR                   Helm chart build output directory.
-                            Default: $ROOT_DIR/build/tmp-helm-$TEST_ID
+  K8S_NAMESPACE:                    K8s Namespace for executing Helm Chart tests
+                                    Default: ack-system-test-helm
+  HELM_CHART_NAME:                  Name of Helm Chart
+                                    Default: ack-<$AWS_SERVICE>-controller
+  SERVICE_CONTROLLER_SOURCE_PATH:   Path to the ACK service controller. Helm Chart under test are present at $SERVICE_CONTROLLER_SOURCE_PATH/helm
+  AWS_SERVICE_DOCKER_IMG:          Name of the controller image which will be used during release test
+  AWS_REGION:                      AWS Region to use for installing ACK service controller. Default: us-west-2
+  AWS_ACCOUNT_ID:                  AWS Account ID to use for installing ACK service controller.
 "
 
-SERVICE="$1"
-RELEASE_VERSION="$2"
-K8S_NAMESPACE="ack-system-test-helm"
-
-if [ -z "$TMP_DIR" ]; then
-    TEST_ID=$(uuidgen | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
-    TMP_DIR=$ROOT_DIR/build/tmp-helm-$TEST_ID
+if [ $# -ne 1 ]; then
+    echo "AWS_SERVICE is not defined. Script accepts one parameter, <AWS_SERVICE> to install, validate and uninstall Helm chart for service controller" 1>&2
+    echo "${USAGE}"
+    exit 1
 fi
 
-echo "testing Helm release for $SERVICE for release version $RELEASE_VERSION."
-mkdir -p $TMP_DIR
-
-trap "clean_up" EXIT
-
-# We need to do this to prevent "unable to open go.mod" errors from
-# ack-generate...
-pushd $ROOT_DIR 1>/dev/null
-
-ACK_GENERATE_IMAGE_REPOSITORY="aws-controllers-k8s" \
-    ACK_GENERATE_SERVICE_ACCOUNT_NAME="ack-$SERVICE-controller-helm-test" \
-    ACK_GENERATE_OUTPUT_PATH="$TMP_DIR" \
-    K8S_RBAC_ROLE_NAME="ack-$SERVICE-controller-helm-test" \
-    $CODE_GENERATOR_SCRIPTS_DIR/build-controller-release.sh "$SERVICE" "$RELEASE_VERSION"
-
-popd 1>/dev/null
-
-kubectl create namespace "$K8S_NAMESPACE" --dry-run=client -o yaml | kubectl apply -f -
-
-pushd $TMP_DIR/helm 1>/dev/null
-
-echo -n "installing the helm chart for ack-$SERVICE-controller in namespace $K8S_NAMESPACE ... "
-helm install --namespace "$K8S_NAMESPACE" ack-$SERVICE-controller-helm-test . 1>/dev/null || exit 1
+# SET VARIABLES FOR EXECUTING HELM TESTS
+
+## AWS_SERVICE
+AWS_SERVICE=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+## K8S_NAMESPACE
+K8S_NAMESPACE=${K8S_NAMESPACE:-"ack-system-test-helm"}
+## HELM_CHART_NAME
```

Comment:
Any reason for these comments?

Reply:
Just my personal preference. Added them for more clarity in readability. 
- Easy to locate env variables in the bash script.
- Easy to read their assignments separately when they are not on consecutive lines.

Reply:
Actually, I don't think these code comments with just the variable name are providing more clarity... they are just repeating the variable name.

However, code comments such as this:

```bash
# Source code for the controller will be in a separate repo, typically in
# $GOPATH/src/github.com/aws-controllers-k8s/$AWS_SERVICE-controller/
```

**do** offer clarity...